### DISCUSSION TITLE
Start packages via cli?

### DISCUSSION THREAD

**Discussion by short-circuit** [LykosAI/StabilityMatrix#1052] [Q&A] [answered]
Is there any way to start packages via cli directly without having to use the UI?
This is useful and mostly needed when remoting into a machine that has everything set up with StabilityMatrix.

**Comment by mohnjiles** [comment, marked as answer]
You can launch SM with the `--launch-package {PackageName}` argument, for example `--launch-package ComfyUI`, though this will still show the SM UI afterwards.
